content/future.html

```diff
@@ -2,104 +2,133 @@ <h1 id="future"> Future </h1>
+</p>
+<div class="half second">
+  <span class="graph-title">Post-grad contentness</span>
+  <div id="post-contentness"></div>
+</div>
+
+<div class="vbx2"></div>
+<h3> Full-Time Compensation </h3>
+<p class="half">
+  <span class="big">
+    Compensation for the first year of full-time work for respondants will be
+    a median of $260,000 and average of $284,748 CAD.
+  </span>
+  The exchange rate used for this question and the ones that follow was 1.3
+  CAD per 1 USD. That rate was accurate as the time the survey was shared,
+  but the rate is 1.2 as of writing.
   <br /><br />
-  It's important to note that there were students from each family income bracket
-  who graduated debt-free.
-  All students with family income higher than $200,000 graduated debt-free.
+  Since 77% of respondants will be relocating in the US, we can estimate
+  the new median and average earnings to actually be $244,600 and $267,882 CAD.
+  <!-- 260k*(1 + 0.77(1.2/1.3 - 1)). -->
 </p>
 <div class="half second">
-  <span class="graph-title">Debt per family income bracket</span>
-  <div id="debt"></div>
+  <span class="graph-title">Total First-Year Compensation as New Grad</span>
+  <div id="ft-total"></div>
 </div>
 
 <div class="vbx2"></div>
+<p class="half">
+  <span class="big">
+    The hourly wage / salary will be a median and average of $81 and
+    $105 CAD respectively.
+  </span>
+  We can again estimate this to be $76 and $99 CAD with today's exchange rate.
+</p>
+<div class="half second">
+  <span class="graph-title">Hourly Wave as New Grad</span>
+  <div id="ft-hourly"></div>
+</div>
 
-<h3> Advice </h3>
-<p class="full">
+<div class="vbx2"></div>
+<p class="half">
   <span class="big">
-    "Figure out which advice applies best to you and ignore the rest."
+    There's a huge discrepency in profit sharing for new grads,
+    with the median and average being $32,500 and $95,392 CAD.
   </span>
-  The following is a summary of the advice shared by students with the
-  intention of being shared with both prospective and current SE students.
-  Many of the sentences
-  are directly quoted or rephrased from the original advice.
-  <br />
-  <p class="quote">
-  Your success should not be defined by others. Just because someone else is better than you at
-  something, it does not mean that you're failing. Instead, be open and realize
-  you can learn something from everyone. Realize that your peers' successes
-  should be motivation rather than a reason to put yourself down.
-  <br /><br />
-  Find value in modesty, empathy, and courage. Learn to develop good habits
-  early on, and budget your time based on your options and priorities. These
-  skills are indispensable during university. Find a hobby you love. That
-  hobby just might be software engineering, or it could be something completely
-  different like drawing video game characters.
+  There are extreme outliers that skewed the results greatly.
```

Comment:
it would be interesting to list the outliers (or do the calculations without the outliers?); tho they are visible on the histogram.

Reply:
Yeah, I think my intention with the text is to summarize the important info. I imagine students reading will want to know what they can generally expect financially when graduating from SE. The outliers exist on the graph so listing the exact dollar amounts isn't very useful.